src/components/AgreementSections/CoveredEResourcesList.js

```diff
@@ -106,49 +106,78 @@ export default class CoveredEResourcesList extends React.Component {
     date ? <FormattedUTCDate value={date} /> : '-'
   )
 
-  renderExportDropdown = () => (
-    <Dropdown
-      onToggle={() => this.setState(prevState => ({ exportDropdownOpen: !prevState.exportDropdownOpen }))}
-      open={this.state.exportDropdownOpen}
+  renderDropdownButton = (disabled) => (
+    <DropdownButton
+      data-role="toggle"
+      disabled={disabled}
     >
-      <DropdownButton data-role="toggle">
-        <FormattedMessage id="ui-agreements.eresourcesCovered.exportAs" />
-      </DropdownButton>
-      <DropdownMenu data-role="menu">
-        <FormattedMessage id="ui-agreements.eresourcesCovered.exportAsJSON">
-          {exportAsJson => (
-            <Button
-              aria-label={exportAsJson}
-              buttonStyle="dropdownItem"
-              id="clickable-dropdown-export-eresources-json"
-              onClick={() => {
-                this.setState({ exportDropdownOpen: false });
-                this.props.onExportEResourcesAsJSON();
-              }}
-            >
-              <FormattedMessage id="ui-agreements.eresourcesCovered.json" />
-            </Button>
-          )}
-        </FormattedMessage>
-        <FormattedMessage id="ui-agreements.eresourcesCovered.exportAsJSON">
-          {exportAsKbart => (
-            <Button
-              aria-label={exportAsKbart}
-              buttonStyle="dropdownItem"
-              id="clickable-dropdown-export-eresources-kbart"
-              onClick={() => {
-                this.setState({ exportDropdownOpen: false });
-                this.props.onExportEResourcesAsKBART();
-              }}
-            >
-              <FormattedMessage id="ui-agreements.eresourcesCovered.kbart" />
-            </Button>
-          )}
-        </FormattedMessage>
-      </DropdownMenu>
-    </Dropdown>
+      <FormattedMessage id="ui-agreements.eresourcesCovered.exportAs" />
+    </DropdownButton>
   )
 
+  renderExportDropdown = () => {
+    const disabled = this.props.eresourcesFilterPath === 'dropped' || this.props.eresourcesFilterPath === 'future';
+    return (
+      <Dropdown
+        onToggle={() => this.setState(prevState => ({ exportDropdownOpen: !prevState.exportDropdownOpen }))}
+        open={this.state.exportDropdownOpen}
+      >
+        {disabled ?
+          <Tooltip
+            id="covered-eresources-export-tooltip"
+            placement="top"
+            text={<FormattedMessage id="ui-agreements.eresourcesCovered.exportButton.tooltip" />}
+          >
+            {({ ref, ariaIds }) => (
+              <div
+                ref={ref}
```

Comment:
You should be setting the ref and aria attribute [on the DropdownButton itself](https://github.com/folio-org/stripes-components/blob/78d983eea6473b92116f8c7bfac3a85c1853068e/lib/DropdownButton/DropdownButton.js#L14), and not on a wrapper `div`.

Reply:
I tried that first but I couldn't  get it working. The tooltip doesn't show up if I write the following
```
<Tooltip
            id="covered-eresources-export-tooltip"
            placement="top"
            text={<FormattedMessage id="ui-agreements.eresourcesCovered.exportButton.tooltip" />}
>
            {({ ref, ariaIds }) => (
               <DropdownButton
                  data-role="toggle"
                  disabled={disabled}
                  ref={ref}
                  aria-describedby={ariaIds.text}
               >
                   <FormattedMessage id="ui-agreements.eresourcesCovered.exportAs" />
              </DropdownButton>
           )}
</Tooltip>
```

Reply:
So the key things are setting the aria- and ref props correctly. If you've confirmed the aria-describedby is being set, then that leaves checking ref.

If you set `ref={ref}` on DropdownButton, then your ref is for the React component itself. You need it to be on the rendered `<button>` (lowercase). So if you look at DropdownButton's code, you see it passes down most props to Button. If you check Button's code, you'll see it allows you to set the ref of the rendered `<button>` if you give it a `buttonRef` prop.

So that'd be the next thing I'd try.

Reply:
adding a `buttonRef` prop to the DropdownButton doesn't work either

Reply:
Hmm, looks like this is actually an issue with [the behaviour of `disabled` Buttons not firing events](https://github.com/folio-org/stripes-components/blob/master/lib/Button/Button.css#L218). This is an issue for us since `Tooltip` relies on the `<button>` [sending `mouseover` events to decide whether to show the tooltip or not](https://github.com/folio-org/stripes-components/blob/master/lib/Tooltip/Tooltip.js#L57).

@rasmuswoelk, do you have a suggestion on what the best course of action is here? Should we go ahead and just wrap the `DropdownButton` in a div like it is, or something else? Turn on `pointer-events` in our CSS for this instance?

Reply:
@doytch Hmm. Good question.

In general, we need Tooltips to be accessible both on hover and focus to comply with the accessibility standards. So when we make the button disabled, it's no longer possible to focus it so in that case, it's actually the intended behavior here (the tooltip not showing on disabled buttons).

Could we solve this by placing an`<Icon tabIndex={0} />` next to the dropdown, which will render the tooltip and have that provide this information for the user?

![image](https://user-images.githubusercontent.com/640976/69729550-c9c85400-1126-11ea-8db2-e7b37bca3a0f.png)

Reply:
Hmmm, that looks pretty nice...

@CalamityC lets go with this approach. It even has the benefit of cleaning up the render code!